File: PYTHON/boca/resolvidos cpp/talita/aumento.py
salario_antigo = float(input())


def get_percentual(salario): 

    if 0<salario<=400.00: 
        percentual = 15/100
        return(percentual) 
    elif 400.01<=salario<=800.00: 
        percentual = 12/100
        return(percentual)
    elif 800.01<=salario<=1200.00: 
        percentual = 10/100
        return(percentual)
    elif 1200.01<=salario<=2000.00: 
        percentual = 7/100
        return(percentual)
    elif 2000.00<salario: 
        percentual = 4/100
        return(percentual)
    
percentual = get_percentual(salario_antigo)
print(percentual)

print(f"Novo salario: {salario_antigo +(salario_antigo*percentual)}")
print(f"Reajuste ganho: {(salario_antigo*percentual)}")
print(f"Em percentual: {int(percentual*100)}%")

File: PYTHON/boca/resolvidos /mauricio/aumento.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main(){
	int per;
	double sal;
	cin >> sal;
	sal = round(sal*100)/100;
	if(sal>=0 && sal<=400){
		per=15;
	}
	else if(sal>400 && sal<=800){
		per=12;
	}
	else if(sal>800 && sal<=1200){
		per=10;
	}
	else if(sal>1200 && sal<=2000){
		per=7;
	}
	else if(sal>2000){
		per=4;
	}
	double reaj = round((sal*(per*0.01))*100)/100;
	sal = sal + reaj;
	cout << fixed << setprecision(2)<<"Novo salario: "<< sal << "\nReajuste ganho: " << reaj << "\n" << "Em percentual: " << per << "%\n";

}
